### PR TITLE
gr-uhd: Adding missing Python binding for uhd::tune_request_t::policy_t.

### PR DESCRIPTION
**Pull request by Codey-McCodeface**
Fix for issue: #4640

### CODE REVIEW

**Comment by mormj** [comment]
@Codey-McCodeface - Thank you for the contribution.  Two simple things to get it to run through the CI tests:

1) Please sign off your commit.  GNU Radio adheres to the DCO (https://wiki.gnuradio.org/index.php/Development#What.27s_this_Developer_Certificate_of_Origin_.28DCO.29.3F) -- this can be done simply by `git commit --amend -s`, then force push the amended commit
2) The clang-format check is failing - the lazy approach here is just run it on the whole codebase: `find ./ -iname *.h -o -iname *.cc | xargs clang-format -i` from the top level source directory

Please let me know if you run into any issues.

**Comment by Codey-McCodeface** [comment]
@mormj Thanks. I think it is all set.

File: gr-uhd/python/uhd/bindings/uhd_types_python.cc [hunk 1]
```diff
@@ -141,8 +141,8 @@ void bind_uhd_types(py::module& m)
 
     // ::uhd::tune_request_t
     using tune_request_t = ::uhd::tune_request_t;
-    py::class_<tune_request_t>(m, "tune_request_t")
-        .def(py::init<double, double>())
+    py::class_<tune_request_t> tune_request(m, "tune_request_t");
+    tune_request.def(py::init<double, double>())
```
Comment (willcode) [inline gr-uhd/python/uhd/bindings/uhd_types_python.cc:145]: What is this change for?

Comment (mormj) [reply gr-uhd/python/uhd/bindings/uhd_types_python.cc:145]: usually the class definition gets broken up that way to shove an enum in with the class, but nothing is being added here.  the change is not harmful, but also not necessary

Comment (Codey-McCodeface) [reply gr-uhd/python/uhd/bindings/uhd_types_python.cc:145]: I needed to reference the tune_request python class to put the policy_t enum within it. I was trying to be consistent with the uhd hierarchy.

I now see that other uhd enums are just defined at the module level. Would this be preferred over nesting it within tune_request?

Comment (mormj) [reply gr-uhd/python/uhd/bindings/uhd_types_python.cc:145]: Yes, what you did makes sense - that is indeed the proper way to get the enum nested under the class name in the bindings.

**Review by mormj — APPROVED** [review]

**Pull request merged**